template/README.md

```diff
@@ -26,3 +26,92 @@ For jest you will need to add this to your package.json
     }
 }
 ```
+
+## Implementation Notes
```

Comment:
This template will be used for both SPA's and shared UI's. Can you adjust this header to be "Shared UI Implementation Notes"?